src/tests/ftest/io/nvme_pool_capacity.yaml

```diff
@@ -0,0 +1,49 @@
+server_manager:
+  srv_timeout: 500
+hosts:
+  test_servers:
+    - server-A
+    - server-B
+  test_clients:
+    - client-C
+    - client-D
+timeout: 5000
+server_config:
+  name: daos_server
+  servers:
+      nr_xs_helpers: 1
+      log_file: daos_server0.log
+      bdev_class: nvme
+      bdev_list: ["0000:81:00.0"]
+      scm_class: dcpm
+      scm_list: ["/dev/pmem0"]
+      scm_mount: /mnt/daos0
+pool:
+    mode: 146
+    name: daos_server
+    svcn: 1
+    control_method: dmg
+ior:
+    no_parallel_job: 10
+    clientslots:
+      slots: 2
+    test_file: daos:testFile
+    repetitions: 1
+    daos_destroy: False
+    iorflags:
+          ior_flags:
+            - "-w -r -R -k -G 1"
+          ior_api:
+            - DAOS
+          obj_class:
+            - "SX"
+    ior_test_sequence:
```

Comment:
Don't we need !mux here to test these sizes?

Reply:
No... We are trying a test case sequence... No muxing needed.